helper/schema/schema_test.go

```diff
@@ -2500,7 +2500,7 @@ func TestSchemaMap_Validate(t *testing.T) {
 			},
 		},
 
-		// Good, because the var is not set and that error will come elsewhere
+		// #1 Good, because the var is not set and that error will come elsewhere
```

Comment:
Noooooo shoulda just made them K/V with test names if you were already touching everything. :grinning: